src/main/java/com/incognia/common/ReasonCode.java

```diff
@@ -44,7 +44,12 @@ public enum ReasonCode {
   DEVICE_INTEGRITY("device_integrity"),
   REPORT("report"),
   TRUSTED_LOCATION("trusted_location"),
-  MULTI_DEVICE_ACCOUNT("multi_device_account");
+  MULTI_DEVICE_ACCOUNT("multi_device_account"),
+  CUSTOM_POS_ATM_FRAUD("custom_pos_atm_fraud"),
+  CUSTOM_ORDER_FLOW_FRAUD("custom_order_flow_fraud"),
```

Comment:
@gaabs

Reply:
I believe it can become a problem because we expose what our clients are reporting to us, it might not always be something public.

About the breaking change, what would be the problem with that? The client will need to update the library anyway.